chapters/intro/main.tex

```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
-
-The remainder of this chapter is as follows: Section~\ref{sec:thesis} sets out
+The remainder of this chapter is as follows: Section~\ref{sec:objective} states
+the primary objective of the thesis; Section~\ref{sec:structure} sets out
 the structure of this thesis and its chapters; Section~\ref{sec:novel} outlines
 the novel contributions of the thesis; Section~\ref{sec:dev} provides an
 overview of the best practices used in developing software for the research
 presented in this thesis, as well as signposting the software projects
 themselves.
 
 
-\section{Thesis structure}\label{sec:thesis}
+\section{Thesis objective}\label{sec:objective}
+
+The primary purpose of this thesis is to utilise machine learning to better
+understand a healthcare population. This purpose is achieved through the novel
+methods for clustering initialisation and algorithm evaluation presented herein.
+The co-sponsors of this project, NHS Wales Cwm Taf Morgannwg University
+Health Board (UHB), are seeking to reveal new insights into their patient
+population through the use of machine learning. In the context of machine
+learning, the term `utilisation' typically refers to the application of some
+existing machine learning apparatus to a dataset. This thesis considers a more
+nuanced definition. Here, `utilisation' is considered as the culmination of
+three parts: \emph{creation}, \emph{evaluation}, and \emph{application}. Each of
+these components is essential to properly utilising machine learning, and the
+process is comparable to honing a craft.
+
+First, a tool is fashioned for a particular purpose, as a machine learning
+method would be for a problem --- or class thereof. Once a prototype has been
+created, the tool must be evaluated and adjusted. This process improves both the
+skilled use of the tool and the tool itself. Evaluating a machine
+learning method extensively is analogous to this process. With a refined tool
+and skilled hand, expertise in that craft can be demonstrated by applying it for
+its intended purpose; this is the final (and often only) stage when utilising
+machine learning in the real world.
+
+\ctmuhb\ are particularly concerned with understanding the operational
+characteristics of their patients presenting chronic obstructive pulmonary
+disease (COPD). COPD is a respiratory condition with known links to deprivation,
+and often presents as a comorbidity, i.e. in concurrence with at least one other
+condition. These affiliations make living with and treating COPD inherently
+difficult, emphasising the importance of studying it closely.
+
+This thesis considers how clustering can be utilised to better understand the
+COPD population treated by \ctmuhb. In order to carry out this utilisation, this
+thesis incorporates three themes from modern OR: algorithm evaluation,
+clustering and segmentation, and operational healthcare modelling. These
+seemingly disparate themes are presented as a triptych, where each theme
```

Comment:
I had to google `triptych`, it's [as I expected -- you're good with them there words] apparently fine but perhaps `triplet` ?